src/custom/pages/App/AppMod.tsx

```diff
@@ -82,13 +100,22 @@ const Marginer = styled.div`
 // }
 
 export default function App(props?: { children?: ReactNode }) {
+  const [bgBlur, setBgBlur] = useState(false)
+  const location = useLocation()
+  useEffect(() => {
+    if (location.pathname.length > 1 && location.pathname !== '/swap') {
+      setBgBlur(true)
+    } else {
+      setBgBlur(false)
+    }
```

Comment:
Refactoring suggestion:

```suggestion
    setBgBlur(location.pathname.length > 1 && location.pathname !== '/swap')
```

Although, not sure should keep this logic. I'll wait for everyone else's input regarding the blur on swap page.

Reply:
@maria-vslvn i think Leandro suggestion is good